.pre-commit-config.yaml

```diff
@@ -5,17 +5,17 @@ repos:
     rev: 21.5b1
     hooks:
       - id: black
-        files: ^backend/
+        files: ^backend/kesaseteli
```

Comment:
I think we can let pre-commit runs on the backend folder and move the `setup.cfg` to a shared location (e.g backend/shared/precommit-setup.cfg)